### ISSUE TITLE
Using reinforced glass on a table frame makes a metal table

### ISSUE THREAD

**Issue by cacogen** [tgstation/tgstation#49683]
Issue reported from  Round ID: 131398 (/tg/Station Manuel [ENGLISH] [US-WEST] [ROLEPLAY] [99% LAG FREE])

Reporting client version: 513.1508

Obviously using plasteel makes a reinforced metal table. We don't have sprites for reinforced glass tables, but it'd be more appropriate if glass made a glass table perhaps with "reinforced glass table" as the name and the equivalent increase in strength that the metal equivalent has.

**Comment by TemporalOroboros** [comment]
Cardboard can also make a metal table.

**Comment by RaveRadbury** [comment]
Using wood on a metal table frame does the same.

**Comment by NamelessFairy** [comment]
I've tested this on local and it doesn't appear to be occurring anymore?
![image](https://user-images.githubusercontent.com/40036527/226111562-abdfc124-d2f0-4984-b91c-d65fcb77b351.png)

**Comment by Sealed101** [comment]
i think most of then were covered nowadays, but some additional coverage wouldn't hurt

**Comment by Sealed101** [comment]
yeah no there're still sheet types with no table variant, not even a generic greyscaled one. all white tables in the image are those, they default their description to `A square piece of nothing on four legs. It can not move.`

![image](https://github.com/tgstation/tgstation/assets/75863639/6528c7ec-6071-4c97-947f-e1f11948bc49)

https://github.com/tgstation/tgstation/blob/65b168af718b69b79b4762e619a3df751a4ff03f/code/game/objects/structures/table_frames.dm#L35-L64

carpets are `stack/tile/carpet` and not all of them have a unique type so they default to the stock carpet's `tableVariant`. this is mostly an issue of sprites though
